cylc/flow/task_pool.py

```diff
@@ -1581,6 +1546,10 @@ def spawn_task(
                     f"[{itask}] won't run: depends on a task beyond "
                     f"the stop point ({self.stop_point})"
                 )
+                # TODO: future trigger overrun tasks were previously kept in
+                # the hidden task pool (for partially satisfied tasks) and
+                # ignored. But perhaps they should cause a stall?
```

Comment:
TODO

What's a "future trigger overrun task"?

Can put up an issue for this and add the link into the comment.

Reply:
I'll have to remind myself again, at this point 😬

Reply:
```
foo
foo[+P1] & bar => baz
```
In the final cycle point, bar will want to spawn baz, but that would stall the workflow because baz also depends on foo beyond the final point.

I think it's fine to simply not spawn baz. 

I've add this as a comment, and removed the unnecessary terminology ("future trigger overrun").